packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -234,10 +234,21 @@ export function makeFontFacesFormData( font ) {
 }
 
 export async function batchInstallFontFaces( fontFamilyId, fontFacesData ) {
-	const promises = fontFacesData.map( ( faceData ) =>
-		fetchInstallFontFace( fontFamilyId, faceData )
-	);
-	const responses = await Promise.allSettled( promises );
+	const responses = [];
+
+	// Uses the same response format as Promise.allSettled, but executes requests in sequence to work around a race condition
+	// that can cause an error when the fonts directory doesn't exist yet.
```

Comment:
Multi-line comment format per coding standards. Made the lines the same length because it looks nicer. 

```suggestion
	/*
	 * Uses the same response format as Promise.allSettled, but executes requests in sequence to work 
	 * around a race condition that can cause an error when the fonts directory doesn't exist yet.
	 */
```

Reply:
Thank you! I will try to remember this one for next time.